app.rb

```diff
@@ -1,8 +1,34 @@
 require 'sinatra/base'
+require './lib/game'
+require './lib/player'
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+  
+  get '/' do
+    erb :index
   end
 
+  post '/option' do
+    $player = Player.new(params[:player_1_name])
```

Comment:
could get rid of the global variable by adding some self methods in game.rb 
```
def self.create(player)
   @game = Game.new(player)
 end

 def self.instance
   @game
 end
```

you could then change app.rb to have a before 

```
before do
  @game = Game.instance
end
```

and include - 
```
player = Player.new(params[:player_1_name])
@game = Game.create(player)
```

you then shouldn't need the global variables